marconi-chain-index/app/Experimental.hs

```diff
@@ -1,28 +1,29 @@
 {-# LANGUAGE OverloadedStrings #-}
 
-import Data.Void (Void)
+import Control.Monad.Except (runExceptT)
 import Marconi.ChainIndex.CLI qualified as Cli
-import Marconi.ChainIndex.Experimental.Indexers qualified as Indexers
-import Marconi.ChainIndex.Experimental.Indexers.MintTokenEvent qualified as Mint
-import Marconi.ChainIndex.Utils qualified as Utils
+import Marconi.ChainIndex.Experimental.Indexers (buildIndexers)
+import Marconi.ChainIndex.Experimental.Indexers.Utxo qualified as Utxo
+import Marconi.ChainIndex.Experimental.Runner qualified as Runner
+import Marconi.Core.Experiment qualified as Core
 import System.Directory (createDirectoryIfMissing)
-import System.FilePath ((</>))
 
 main :: IO ()
 main = do
   o <- Cli.parseOptions
   createDirectoryIfMissing True (Cli.optionsDbPath o)
-
   let socketPath = Cli.optionsSocketPath $ Cli.commonOptions o
       networkId = Cli.optionsNetworkId $ Cli.commonOptions o
-  securityParam <- Utils.toException $ Utils.querySecurityParam @Void networkId socketPath
-  indexers <-
-    sequence
-      [ fmap snd $ Indexers.utxoWorker (Cli.optionsDbPath o </> "utxo.db") securityParam
-      , fmap snd $ Mint.mintBurnTokenEventWorker (Cli.optionsDbPath o </> "minttokenevent.db")
-      ]
-
-  Indexers.runIndexers
+  mindexers <-
+    runExceptT $
+      buildIndexers
+        (Core.CatchupConfig 5000 100)
```

Comment:
Use vars for those magic numbers?

Reply:
Ideally, I would add it as CLI options in a next PR, I can use var instead at the moment.